xla/tsl/util/onednn_threadpool.h

```diff
@@ -93,6 +93,14 @@ class OneDnnThreadPool : public threadpool_iface {
     return (eigen_interface_->CurrentThreadId() != -1) ? true : false;
   }
   virtual uint64_t get_flags() const override { return ASYNCHRONOUS; }
+#ifdef ENABLE_ONEDNN_ASYNC
+  virtual void wait() override {
+    // wait() method for synchronous execution is basically a no-op.
+    // But we need to implement it to satisfy the interface.
+    // This is the requirement of the new experimental async runtime support
+    // in oneDNN.
+  }
```

Comment:
Nit: Let's move the comment outside of the function to make it clear that it's empty.
```suggestion
  // wait() method for synchronous execution is basically a no-op.
  // But we need to implement it to satisfy the interface.
  // This is the requirement of the new experimental async runtime support
  // in oneDNN.
  virtual void wait() override {}
```

Reply:
Done.